src/test/java/AuthenticationFormTest.java

```diff
@@ -0,0 +1,56 @@
+import core.TestBase;
+import helpers.DataProviders;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pages.StaticRegistrationPage;
+
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.open;
+
+public class AuthenticationFormTest extends TestBase {
+
+    @BeforeTest
+    public void Setup() {
+        open("/login");
+    }
+
+    @Test(dataProvider = "loginData", dataProviderClass = DataProviders.class)
+    public void authenticationTest(String login, String pswd) {
+        if (login.equals(" ")) {
```

Comment:
taking two comments above in account you won't have so many if (actually only one will be left -> check on validation message)